docs/webusb.md

```diff
@@ -0,0 +1,35 @@
+Zephyr.js API for WebUSB
+========================
+
+* [Introduction](#introduction)
+* [Class: WebUSB](#class-webusb)
+* [API Documentation](#api-documentation)
+* [Sample Apps](#sample-apps)
+
+Introduction
+------------
+The WebUSB module supports advertising the device as a WebUSB device when the
```

Comment:
Web USB, not WebUSB (yeah web is inconsistent, but spaces is preferred)

Reply:
But the spec is called [WebUSB](https://wicg.github.io/webusb/)...